src/main/java/seedu/address/ui/StudentCard.java

```diff
@@ -15,6 +15,7 @@
 public class StudentCard extends UiPart<Region> {
 
     private static final String FXML = "StudentListCard.fxml";
+    private static final String TELEGRAM_PREFIX = "@";
```

Comment:
Might be good if we remove the telegram prefix, since user has to type @ for email. So we might want to keep the design consistent by making them type @ for telegram handle

Reply:
Resolved by sticking to using TELEGRAM_PREFIX = "@"